### PR TITLE
Bump image for ci-kubernetes-e2e-kops-aws-newrunner

### PR DESCRIPTION
**Pull request by justinsb**
We want the --kops-version support in kubetest

### CODE REVIEW

**Comment by justinsb** [comment]
Wondering whether I should just use `:latest` for this job until I get it working; it is pretty clearly marked as "-newrunner" so I think/hope people won't be misled!

**Comment by justinsb** [comment]
(logs from failed run https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-kops-aws-newrunner/1, though it did work locally for me from master :-) )

**Comment by krzyzacy** [comment]
yeah maybe just use `:latest-master` until it's stablized

**Comment by justinsb** [comment]
Thanks - switching to `gcr.io/k8s-testimages/kubekins-e2e:latest-master`.

File: prow/config.yaml [hunk 1]
```diff
@@ -15330,7 +15330,7 @@ periodics:
         value: /etc/aws-ssh/aws-ssh-private
       - name: JENKINS_AWS_SSH_PUBLIC_KEY_FILE
         value: /etc/aws-ssh/aws-ssh-public
-      image: gcr.io/k8s-testimages/kubekins-e2e:v20171023-364289b6-master
+      image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
```
Comment (krzyzacy) [inline prow/config.yaml]: can you add `imagePullPolicy: Always` as well?

Comment (justinsb) [reply prow/config.yaml]: Good point - thanks!  Updated!

Out of interest / penance - do we have a tester / linter for bad prow configs I could add this check to?

**Comment by justinsb** [comment]
But I also see there was an across-the-board image update, and this test is almost green anyway now :-) https://k8s-testgrid.appspot.com/google-aws#kops-aws-newrunner

**Comment by krzyzacy** [comment]
:+1: 
yeah we probably can add some tests to prow/config/config_test.go

/lgtm

**Comment by justinsb** [comment]
Thanks & thanks for the pointer @krzyzacy - added a quick test in #5225

**Pull request merged**